tests/test_utils.py

```diff
@@ -289,7 +289,7 @@ def test_make_file_id_570(self):
     def test_make_file_id_605(self):
         """https://github.com/OCR-D/core/pull/605"""
         mets = OcrdMets.empty_mets()
-        f = mets.add_file('1:!GRP', ID='FOO_0001', pageId='phys0001')
+        f = mets.add_file('2:!GRP', ID='FOO_0001', pageId='phys0001')
```

Comment:
We should probably disallow filegroups starting with a number because the resulting ID might lead to an invalid `xsd:ID` because they mustn't start with a number.

Reply:
Yes, if #746 kicks in, such tests should all fail...